server/config.go

```diff
@@ -53,42 +59,32 @@ func (c *Configuration) Validate() error {
 		return ErrServiceNameMissing
 	}
 
-	// Validate (basic checks for existence of capabilities)
-	if c.Capabilities == nil {
-		return ErrCapabilitiesMissing
-	} else if len(c.Capabilities.BsvAlias) == 0 {
+	if c.BSVAliasVersion == "" {
 		return ErrBsvAliasMissing
-	} else if len(c.Capabilities.Capabilities) == 0 {
+	}
+
+	if c.callableCapabilities == nil || len(c.callableCapabilities) == 0 {
 		return ErrCapabilitiesMissing
 	}
 
 	return nil
 }
 
 // IsAllowedDomain will return true if it's an allowed paymail domain
-func (c *Configuration) IsAllowedDomain(domain string) (success bool) {
-
+func (c *Configuration) IsAllowedDomain(domain string) bool {
 	if c.PaymailDomainsValidationDisabled {
-		success = true
-		return
+		return true
 	}
 
-	// Sanitize the domain (standard)
 	var err error
 	if domain, err = paymail.SanitizeDomain(domain); err != nil {
-		// todo: log the error? This should rarely occur
-		return
+		c.Logger.Warn().Err(err).Msg("failed to sanitize domain")
```

Comment:
Why warning?

Reply:
My initial policy regarding 'warning or error' was as follows:

- Error: if this event occurred due to a bug or incorrect configuration.
- Warning: if this event occurred due to incorrect input data from a user.

Later, I discovered the function p2pDestination in p2p_payment_destination.go. This function is used to check whether a user has provided the correct domain in their paymail. Based on this, I decided to opt for a warning.

However, if you're not convinced, I am willing to change it to Err.